### ISSUE TITLE
The "Copy/Select All" menu is wrongly displayed when the "Autoplay" button from the recommended videos on Youtube is long clicked

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#2474]
**[Affected Versions]:**
- Firefox Reality version 8-33470001 (52a50579)

**[Affected Platforms]:**
- Oculus Go - Android 7.1.2
- HTC Vive Focus Plus - Android 7.1.2

**[Prerequisites]:**
- The Firefox Reality browser is opened.

**[Steps to reproduce]:**
1. Navigate to Youtube website and open a video.
2. Long click on the "Autoplay" button from the recommended videos on the top right.
3. Observe if the "Copy/Select All" menu is displayed.

**[Expected results]:**
- The menu is NOT displayed.

**[Actual results]:**
- The menu is displayed.

**[Notes]:**
- No input is displayed if the "section" is copied and pasted.
- Needed by #1760.
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=1USKz6_sp8CkOpm-ERwc-62OqCT_qYcYS).

**Comment by keianhzo** [comment]
@Softvision-RemusDranca I can't reproduce this anymore, Can you confirm it still happens and that the STRs are still valid?

**Comment by Softvision-RemusDranca** [comment]
I've retested this issue using the same steps and it's no longer reproducible with the Firefox Reality version 10 RC5 (e84d3c9). Now, nothing happens when long clicking the "Autoplay" button from the recommended videos.
Considering this I will close this issue as verified fixed.

Tested on HTC Vive Focus (Android 7.1.2), Oculus Quest (Android 7.1.1), and Pico Neo 2 (Android - 8.1.0) devices.
